concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1393,6 +1395,11 @@ jobs:
 
                   echo "Generating manifests"
 
+                  if [ "${ENABLE_DATADOG}" = "true" ] ; then
+                     MANIFEST_STUBS="${MANIFEST_STUBS}
+                                    ./paas-cf/manifests/cf-manifest/stubs/datadog-nozzle.yml"
```

Comment:
extensions is for anything conditionally added to the main manifest - be it properties, VMs, jobs, whatever. `stubs` looks like another `manifest`